x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/repository_list/repository_list.tsx

```diff
@@ -135,7 +135,7 @@ export const RepositoryList: React.FunctionComponent<RouteComponentProps<MatchPa
     content = (
       <RepositoryTable
         repositories={repositories || []}
-        managedRepository={managedRepository.name}
+        managedRepository={managedRepository && managedRepository.name}
```

Comment:
nit: now we can write it like this `managedRepository={managedRepository?.name}`